pacemaker.spec.in

```diff
@@ -24,6 +24,7 @@
 %global cs_version %(pkg-config corosync --modversion  | awk -F . '{print $1}')
 # It has to be eventually decided whether to use Python2 or Python3
 %global py_site    %{?python_sitearch}%{!?python_sitearch:%(python -c "from distutils.sysconfig import get_python_lib; print(get_python_lib(1))")}
+%global cman_allow (0%{?fedora} > 0 && 0%{?fedora} < 17) || (0%{?rhel} > 0 && 0%{?rhel} < 7)
```

Comment:
This macro is a good idea, but it's not really about allowing CMAN, it's just about providing Fedora/RHEL-specific prerequisites. Many other distros can run CMAN. Maybe call it cman_el or such. I wonder if CentOS, Scientific Linux, etc., match ?rhel.